production/helm/loki/values.yaml

```diff
@@ -429,6 +440,14 @@ loki:
   # --  Optional index gateway configuration
   index_gateway:
     mode: simple
+  # --  Optional bloom gateway configuration
+  bloom_gateway:
+    enabled: false
+    client:
+      addresses: '{{ include "loki.indexGatewayAddress" . }}'
```

Comment:
According to https://grafana.com/docs/loki/latest/operations/query-acceleration-blooms/#enable-query-acceleration-with-blooms this should be address of bloom-gateway, not index-gateway.

Reply:
Oh yes you are right, I'll fix this :(

Reply:
It's changed :)